### DISCUSSION TITLE
Filament firebase authentication

### DISCUSSION THREAD

**Discussion by gbarstys** [filamentphp/filament#10834] [General]
Hello,

Is it possible to connect filament login auth with firebase auth?

**Comment by ArtMin96** [comment]
Filament auth is Laravel's auth. And if you can connect Laravel's auth to Firebase's auth, then you can connect Filament as well.

**Reply by gbarstys** [reply]
Yes. On Laravel sample project I added Firebase auth and everything worked. But when I try to add this to filament it doesn't work. I can't find any example which files I need to change to make it work.

**Reply by ArtMin96** [reply]
In AdminPanelProvider

```php
$panel
    ->login(YourLoginClass::class)
```
